.env.env_var.template

```diff
@@ -20,3 +20,7 @@ export DEFAULT_PAGE_SIZE=5
 export DEFAULT_DATE_FORMAT="DD/MM/YYYY"
 
 export RATELIMIT_STORAGE_URI=memory://
+
+# record bucket variables
+
+export RECORD_BUCKET_NAME=test-record-download
```

Comment:
could leave this empty to reduce confusion - devs will most likely need to change this

Reply:
please update the existing config testing in test_config.py to reflect new changes

Reply:
Updated testing in test_config.py to reflect new changes

Removed value